### PR TITLE
GLEP-81 migration

### PR DESCRIPTION
**Pull request by xmhd**
GLEP 81 migrations for various packages

See: #559

GID + UID are an initial stab, based off what ID's these groups/users have on our existing systems.

Don't merge this until it has been reviewed by multiple people - there may be GID/UID changes required, or even just set to -1.

### CODE REVIEW

**Comment by xmhd** [comment]
Confirmed good as per @alip :+1: 
- aerospike
- zabbix
- logstash
- prometheus

**Comment by xmhd** [comment]
I've looked over all of this, any I'm unsure about I have set to -1 for gid + uid (we can change these later - this just keeps current behaviour but migrates to new packages for handling gid + uid).

Please review when possible @alip @varilci @monsieurp

File: dev-db/metabase-bin/metabase-bin-0.34.0.ebuild [hunk 1]
```diff
@@ -1,10 +1,8 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2019 Gentoo Authors
```
Comment (varilci) [inline dev-db/metabase-bin/metabase-bin-0.34.0.ebuild:1]: 2021?

Comment (xmhd) [reply dev-db/metabase-bin/metabase-bin-0.34.0.ebuild]: fixed

**Comment by xmhd** [comment]
> It looks good to me, we may want one more review from Patrice before merging this. -that's why I am not giving a direct approval-
> 
> I can provide an approval if desired.

there is no rush, we can wait on patrice and alip.

**Pull request merged**